cypress/support/commands.js

```diff
@@ -23,3 +23,7 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
```

Comment:
where do you use this command? remove file if you don't use it

Reply:
nowhere unfortunately. I tried to use it, it doesn't work for me. I forgot to delete this.